### PR TITLE
Completed Reservation Workflow

### PR DESCRIPTION
**Pull request by clee4037**
Summary:
- Tested and connected all API endpoints
- Integrated Steven's TimeSlotList component
- Connected all components and confirmed full workflow is working properly

Reservation Workflow:
- User submits location and date
    - [ ]  We generate geocode (long/lat)
    - [ ]  We query for the closest garages
    - [ ]  We display garages in mapView
- User selects and confirms garage
    - [ ]  We query for all available times
    - [ ]  We display available times in vertical list
- User selects time slot
    - [ ]  We display confirmation page with all details
- User confirms reservation details
    - [ ]  We post to db

Next:
- Implement Redux
- Cleanup code
- Add field validation
- Add styling & daisyUI

### CODE REVIEW

File: src/App.jsx
Comment (bzw24) [inline src/App.jsx:1]: Yea it was missing a closing fragment, good catch.

File: src/containers/UserReservation.jsx
Comment (bzw24) [inline src/containers/UserReservation.jsx:1]: Good that you removed the two imports. They were causing compilation errors.

**Review by bzw24 — APPROVED** [review]
Okay with all the changes

File: src/components/user-reservation/Confirmation.jsx
Comment (stevkim) [inline src/components/user-reservation/Confirmation.jsx:1]: Might need to wrap the reservation && in the return statement -- its js logic which usually needs to be wrapped in curly braces, but idk if it works/bugs out

**Review by stevkim — APPROVED** [review]
Everything looks great, just a quick note on it!

**Pull request merged**